pkg/gui/presentation/branches.go

```diff
@@ -40,11 +42,32 @@ func getBranchDisplayStrings(
 	itemOperation types.ItemOperation,
 	fullDescription bool,
 	diffed bool,
+	viewWidth int,
 	tr *i18n.TranslationSet,
 	userConfig *config.UserConfig,
 	worktrees []*models.Worktree,
 	now time.Time,
 ) []string {
+	checkedOutByWorkTree := git_commands.CheckedOutByOtherWorktree(b, worktrees)
+	showCommitHash := fullDescription || userConfig.Gui.ShowBranchCommitHash
+	branchStatus := BranchStatus(b, itemOperation, tr, now)
+	worktreeIcon := lo.Ternary(icons.IsIconEnabled(), icons.LINKED_WORKTREE_ICON, fmt.Sprintf("(%s)", tr.LcWorktree))
+
+	// Recency usually takes up three characters, plus one for the space
+	availableWidth := viewWidth - 4
```

Comment:
This is only true most of the time; as far as I can see, recency can sometimes be 4 characters (for 10 or 11 months). I don't see what we can do about it, because we only know the width of all columns once rendering is done. Hopefully this will be rare enough that we can just ignore it.

Reply:
That's a good idea; however, I don't really know what I was thinking, there are no recency strings that take up 4 characters, so I opted for keeping this inside the function for simplicity, and just fixing the comment (dfd0e343c06d).